gabbi/case.py

```diff
@@ -352,8 +353,38 @@ def _simple_replacer_regex(key):
 
     def _response_replace(self, message):
         """Replace a content path with the value from a previous response."""
-        return re.sub(self._replacer_regex('RESPONSE'),
-                      self._response_replacer, message)
+        string = re.sub(self._replacer_regex('RESPONSE'),
+                        self._response_replacer, message)
+        try:
+            if string[0] == '{':
+                # Looks like a JSON object.
+                return self._json_loads_coerce_types(string)
+            else:
+                return self.json_loads(string)
```

Comment:
I'm guessing you probably wanted `json.loads` here, not `self.json_loads` (which doesn't exist).

Also, for sake of tidiness we've made the habit of not having JSON processing in this file and instead do most JSON related things in `gabbi.handlers.jsonhandler`, even if it is just utility functions or classmethods that are json-ish, not specifically related to content handlers.